### PR TITLE
KIWI-1590-QA Updates

### PR DESCRIPTION
**Pull request by MohamedHassan101**
### Issue tracking

- [KIWI-1590](https://govukverify.atlassian.net/browse/KIWI-1590)

### PII logging

- [ ] Verified that no PII data is being logged

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[KIWI-1590]: https://govukverify.atlassian.net/browse/KIWI-1590?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/tests/api/BavHappyPath.test.ts [hunk 1]
```diff
@@ -270,7 +270,7 @@ describe("BAV CRI: /abort Endpoint Happy Path Tests", () => {
 
 		const response = await abortPost(sessionId);
 		expect(response.status).toBe(200);
-		expect(response.data).toBe("Session has been aborted");
+		expect(response.data).toBe("Session has already been aborted");
```
Comment (Caitlin-cooling) [inline src/tests/api/BavHappyPath.test.ts:273]: This seems unrelated to the code changes how come it needs to be changed?

Comment (MohamedHassan101) [reply src/tests/api/BavHappyPath.test.ts:273]: Although it was unrelated to this ticket, the test was failing, prompting me to fix it.

Comment (Caitlin-cooling) [reply src/tests/api/BavHappyPath.test.ts:273]: Hmm is it currently failing in main?

Comment (MohamedHassan101) [reply src/tests/api/BavHappyPath.test.ts:273]: It appears that the tests are only failing on this branch , while they are functioning as expected on the main branch. Which leads me to believe that the logic have been changed on this branch.Main is still passing with  `expect(response.data).toBe("Session has been aborted");`

File: src/tests/utils/ApiTestSteps.ts [hunk 2]
```diff
@@ -301,10 +301,15 @@ export function validateJwtToken(jwtToken: any): void {
 	validateRawBody(rawBody);
 }
 
-function validateRawHead(rawHead: any): void {
+async function validateRawHead(rawHead: any): Promise<void> {
```
Comment (Caitlin-cooling) [inline src/tests/utils/ApiTestSteps.ts:304]: Since `validateRawHead` has been changed to `async` do we now need to `await` it where it's called to make sure the test fully executes?

Comment (MohamedHassan101) [reply src/tests/utils/ApiTestSteps.ts:303]: we have an await for `const hashBuffer = await crypto.subtle.digest("SHA-256", msgBuffer);`

Comment (MohamedHassan101) [reply src/tests/utils/ApiTestSteps.ts:303]: we have an await on `validateRawHead` just a few lines above

**Review by Caitlin-cooling — APPROVED** [review]
